test/base/src/jvmTest/kotlin/io/realm/kotlin/test/compiler/set/SetTests.kt

```diff
@@ -32,8 +33,8 @@ import kotlin.test.assertTrue
 class SetTests {
 
     private val supportedPrimitiveTypes = TypeDescriptor.classifiers.keys.filter {
-        // Filter out RealmObject
-        it != RealmObject::class
+        // Filter out RealmObject and MutableRealmInt
```

Comment:
I guess supporting MutableRealmInt in Sets would be weird, and could easily break very fast.